### PR TITLE
[ET-752] [ETP-262] [ET-744] Clean up save_ticket() for Tribe Commerce and fix shared ticket empty indiv capacity bug

### PR DESCRIPTION
**Pull request by sc0ttkclark**
* [ET-752]
* [ETP-262]
* [ET-744]
* [ET-614] Classic Screencast: https://share.getcloudapp.com/04uK26K5 | Block Screencast: https://share.getcloudapp.com/GGuNWeLJ

[ET-752]: https://moderntribe.atlassian.net/browse/ET-752
[ETP-262]: https://moderntribe.atlassian.net/browse/ETP-262
[ET-744]: https://moderntribe.atlassian.net/browse/ET-744
[ET-614]: https://moderntribe.atlassian.net/browse/ET-614

### CODE REVIEW

File: src/Tribe/Commerce/PayPal/Main.php [hunk 1]
```diff
@@ -1214,12 +1214,12 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
-		// Updates if we should show Description
-		$ticket->show_description = isset( $ticket->show_description ) && tribe_is_truthy( $ticket->show_description ) ? 'yes' : 'no';
-
 		/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
 		$tickets_handler = tribe( 'tickets.handler' );
 
+		// Updates if we should show Description
```
Comment (tr1b0t) [inline src/Tribe/Commerce/PayPal/Main.php]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 066144d01f0cb0fe9e17a6c01cf633da</sub>

File: src/Tribe/Commerce/PayPal/Main.php [hunk 2]
```diff
@@ -1334,6 +1334,11 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 		// Makes sure it's an Int after this point
 		$data['stock'] = (int) $data['stock'];
 
+		// The only available value lower than zero is -1 which is unlimited
```
Comment (tr1b0t) [inline src/Tribe/Commerce/PayPal/Main.php]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 1d79f07492bd7ffa9a804cea838513ff</sub>

File: src/Tribe/Commerce/PayPal/Main.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/Commerce/PayPal/Main.php]:
```suggestion
		// Updates if we should show Description.
```

File: src/Tribe/Commerce/PayPal/Main.php (hunk 2 above)
Comment (sc0ttkclark) [reply src/Tribe/Commerce/PayPal/Main.php]:
```suggestion
		// The only available value lower than zero is -1 which is unlimited.
```

**Comment by sc0ttkclark** [comment]
One small fail on this but it's ready for code review (prior to committing ALL the snapshots.

File: readme.txt [hunk 3]
```diff
@@ -117,6 +117,10 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
+= [Capacity] TBD =
+
+* Fix - Make sure when changing unlimited ticket to be a shared capacity ticket that the empty individual capacity gets properly overridden to the shared capacity for Tribe Commerce. [ET-752]
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Fix - Make sure when changing unlimited ticket to be a shared capacity ticket that the empty individual capacity gets properly overridden to the shared capacity for Tribe Commerce. [ET-752]
* Fix - Prevent ticket capacity showing sold out when you have unlimited tickets or RSVPs alongside shared capacity tickets unless it really is sold out. [ET-744]
```

**Comment by bordoni** [comment]
Let's get those Snapshots for ET. This looks good.

**Pull request merged**